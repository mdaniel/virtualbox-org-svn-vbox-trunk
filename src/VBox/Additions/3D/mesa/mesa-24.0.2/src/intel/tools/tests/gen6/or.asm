or(8)           g29<1>UD        g9<4>.xUD       0x00000014UD    { align16 1Q };
or(8)           g43<1>UD        g44<4>UD        1D              { align16 1Q };
or(1)           g28<1>UD        g28<0,1,0>UD    g57<0,1,0>UD    { align1 1N };
or(8)           g10<1>UD        g9<8,8,1>UD     g8<8,8,1>UD     { align1 1Q };
or(16)          g16<1>UD        g14<8,8,1>UD    g12<8,8,1>UD    { align1 1H };
or(1)           g9<1>UD         g0<0,1,0>UD     0x00000800UD    { align1 WE_all 1N };
or.nz.f0.0(8)   null<1>.xUD     g17<4>.xUD      g16<4>.xUD      { align16 1Q };
or.nz.f0.0(8)   null<1>UD       g16<8,8,1>UD    g17<8,8,1>UD    { align1 1Q };
(+f0.0) or(8)   g18<1>UD        g18<8,8,1>UD    0x3f800000UD    { align1 1Q };
or.nz.f0.0(16)  null<1>UD       g28<8,8,1>UD    g30<8,8,1>UD    { align1 1H };
(+f0.0) or(16)  g31<1>UD        g31<8,8,1>UD    0x3f800000UD    { align1 1H };
(+f0.0) or(8)   g22<1>.xUD      g22<4>.xUD      0x3f800000UD    { align16 1Q };
or.nz.f0.0(8)   g8<1>UD         g4<8,8,1>UD     g7<8,8,1>UD     { align1 1Q };
or.nz.f0.0(16)  g12<1>UD        g5<8,8,1>UD     g10<8,8,1>UD    { align1 1H };
or(8)           g4<1>.xUD       g57<4>.xUD      g56<4>.xUD      { align16 1Q };
