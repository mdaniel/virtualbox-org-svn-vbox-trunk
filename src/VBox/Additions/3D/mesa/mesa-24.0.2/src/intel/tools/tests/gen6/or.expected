06 01 60 00 21 0c af 23 20 01 60 00 14 00 00 00
06 01 60 00 21 1c 6f 25 84 05 6e 00 01 00 00 00
06 00 00 00 21 04 80 23 80 03 00 00 20 07 00 00
06 00 60 00 21 04 40 21 20 01 8d 00 00 01 8d 00
06 00 80 00 21 04 00 22 c0 01 8d 00 80 01 8d 00
06 02 00 00 21 0c 20 21 00 00 00 00 00 08 00 00
06 01 60 02 20 04 01 20 20 02 60 00 00 02 60 00
06 00 60 02 20 04 00 20 00 02 8d 00 20 02 8d 00
06 00 61 00 21 0c 40 22 40 02 8d 00 00 00 80 3f
06 00 80 02 20 04 00 20 80 03 8d 00 c0 03 8d 00
06 00 81 00 21 0c e0 23 e0 03 8d 00 00 00 80 3f
06 01 61 00 21 0c c1 22 c0 02 60 00 00 00 80 3f
06 00 60 02 21 04 00 21 80 00 8d 00 e0 00 8d 00
06 00 80 02 21 04 80 21 a0 00 8d 00 40 01 8d 00
06 01 60 00 21 04 81 20 20 07 60 00 00 07 60 00
